packages/components/react-web/src/Uploader/helpers.js

```diff
@@ -0,0 +1,50 @@
+const url = process.env.STORYBOOK_POST_FILE_ENDPOINT;
+const headers = new Headers({
```

Comment:
Love the abstraction here—let's talk about how/if this is generic enough for other usages—I have no idea what the right answer is, just wanna investigate!